external/kafka-0-10/pom.xml

```diff
@@ -49,8 +49,36 @@
     </dependency>
     <dependency>
       <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
```

Comment:
That's good, but what about the other POM file? I know it's not directly related to your change, but it would be a good chance to make these consistent @timvw